prettier.config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
```

Comment:
Copyright?

Reply:
Don't think it makes sense to copyright this file; added anyway. 🤷🏻‍♂️